test/unittests/converters/test_oasis_to_bids_utils.py

```diff
@@ -310,6 +310,55 @@ def test_get_image_with_good_orientation(tmp_path: Path) -> None:
     assert len(image.shape) == 3  # the image dimension should be 3D
 
 
+def test_get_subjects_list_from_data(tmp_path: Path) -> None:
```

Comment:
```suggestion
def test_get_subjects_list_from_data(tmp_path) -> None:
```

In test functions you do not need to define the type of `tmp_path` as it is a pytest fixture (same below). You would need to define types in helper functions if you ever write one though